src/script/event/NotificationService.ts

```diff
@@ -176,7 +176,7 @@ export class NotificationService {
       message = await this.storageService.db.table(StorageSchemata.OBJECT_STORE.EVENTS).where({id: messageId}).first();
     }
 
-    const {notifications} = await this.apiClient.notification.api.getNotifications(clientId);
+    const notifications = await this.apiClient.notification.api.getAllNotifications(clientId);
```

Comment:
What implications does this have? Seems like this is more than a `chore`, isn't it?

Reply:
Nope. It's a util function. The implication is that we get now all notification instead of just the latest 1000.